api/schema/aws_account_resource.go

```diff
@@ -32,7 +32,7 @@ var (
 						Mode: ImmutableAttributeMode,
 					},
 				},
-				"mode": StringResourceAttributeWithMode{
+				"access_mode": StringResourceAttributeWithMode{
```

Comment:
More generally, why is this change necessary?

Reply:
to better match the db field name, and i felt the documentation says it anyways that its access_mode.